tests/hmac.rs

```diff
@@ -1,38 +1,46 @@
-use jsonwebtoken::errors::ErrorKind;
-use jsonwebtoken::jwk::Jwk;
-use jsonwebtoken::{
-    crypto::{sign, verify},
-    decode, decode_header, encode, Algorithm, DecodingKey, EncodingKey, Header, Validation,
-};
 use serde::{Deserialize, Serialize};
 use std::collections::HashMap;
 use time::OffsetDateTime;
 use wasm_bindgen_test::wasm_bindgen_test;
 
+use jsonwebtoken::errors::ErrorKind;
+use jsonwebtoken::jwk::Jwk;
+use jsonwebtoken::{
+    // crypto::{sign, verify},
+    decode,
+    decode_header,
+    encode,
+    Algorithm,
+    DecodingKey,
+    EncodingKey,
+    Header,
+    Validation,
+};
+
 #[derive(Debug, PartialEq, Eq, Clone, Serialize, Deserialize)]
 pub struct Claims {
     sub: String,
     company: String,
     exp: i64,
 }
 
-#[test]
-#[wasm_bindgen_test]
-fn sign_hs256() {
-    let result =
-        sign(b"hello world", &EncodingKey::from_secret(b"secret"), Algorithm::HS256).unwrap();
-    let expected = "c0zGLzKEFWj0VxWuufTXiRMk5tlI5MbGDAYhzaxIYjo";
-    assert_eq!(result, expected);
-}
+// #[test]
+// #[wasm_bindgen_test]
```

Comment:
can we re-enable the wasm tests? are they failing with aws_lc?

Reply:
Ah no, the `sign` and `verify` functions were gone when I picked up the partial PR, but I've re-added them and it works just fine.